nautobot/extras/tasks.py

```diff
@@ -98,7 +98,7 @@ def provision_field(field_id, content_type_pk_set):
 
 
 @nautobot_task
-def process_webhook(webhook_pk, data, model_name, event, timestamp, username, request_id):
+def process_webhook(webhook_pk, data, snapshot, model_name, event, timestamp, username, request_id):
```

Comment:
It *might* be better to add `snapshot` to the end of the args list instead of inserting it into the middle, just for consistency's sake:

```suggestion
def process_webhook(webhook_pk, data, model_name, event, timestamp, username, request_id, snapshot):
```